bench/utils/app.py

```diff
@@ -222,6 +227,18 @@ def get_remote(app, bench_path="."):
 		return contents.splitlines()[0].split()[0]
 
 
+def get_app_name_from_setup(contents: str) -> str:
```

Comment:
```suggestion
def get_app_name_from_setup(contents: str) -> str | None:
```

Or `return ""` or raise some exception at the end after the loop/checks?

Reply:
#1669 0a75989